internal/controllers/reconciliation/controller.go

```diff
@@ -189,6 +189,9 @@ func (c *Controller) reconcileResource(ctx context.Context, prev, resource *reco
 		if current == nil || current.GetDeletionTimestamp() != nil {
 			return false, nil // already deleted - nothing to do
 		}
+		if comp.Annotations["eno.azure.io/deletion-strategy"] == "orphan" {
```

Comment:
nit: Expose these as consts?

Reply:
We might need to reference them from the ask side

Reply:
I have weird opinions about consts lol. Let's chat sometime and hash out how to approach consts across the project :)